src/uwtools/api/jedi.py

```diff
@@ -0,0 +1,54 @@
+"""
+API access to the ``uwtools`` ``jedi`` driver.
+"""
+
+import datetime as dt
+from pathlib import Path
+from typing import Dict, Optional
+
+import uwtools.drivers.support as _support
+from uwtools.drivers.jedi import JEDI as _JEDI
+
+
+def execute(
+    task: str,
+    config: Path,
+    cycle: dt.datetime,
+    batch: bool = False,
+    dry_run: bool = False,
+    graph_file: Optional[Path] = None,
+) -> bool:
+    """
+    Execute a jedi task.
```

Comment:
```suggestion
    Execute a JEDI task.
```

Reply:
I think @christinaholtNOAA and I were reviewing simultaneously and made some similar comments. No pile-on intended!